.github/workflows/code-scanner.yaml

```
@@ -5,6 +5,11 @@ on:
     branches:
       - develop
       - main
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)